api/applications/applicationRouter.js

```diff
@@ -208,29 +205,33 @@ router.get('/profileId/:id', checkApplicationExists, checkRole, (req, res) => {
 
 // create a new user profile and application ticket
```

Comment:
Hi @FatimahSarwar can you please remove all comments unless strictly needed for explanation outside the readme?